cloudnet-modules/cloudnet-cloudflare/src/main/java/de/dytanic/cloudnet/ext/cloudflare/CloudflareAPI.java

```diff
@@ -103,11 +103,11 @@ public static CloudflareAPI getInstance() {
         JsonDocument document = JsonDocument.newDocument(this.readResponse(httpURLConnection));
         httpURLConnection.disconnect();
 
-        this.update(serviceName, statusCode, email, apiKey, zoneId, document);
+        this.update(serviceName, statusCode, email, authenticationMethod, apiKey, zoneId, document);
         return new Pair<>(statusCode, document);
     }
 
-    public Pair<Integer, JsonDocument> deleteRecord(String email, String apiKey, String zoneId, String recordId) {
+    public Pair<Integer, JsonDocument> deleteRecord(String email, CloudflareConfigurationEntry.AuthenticationMethod authenticationMethod, String apiKey, String zoneId, String recordId) {
         Preconditions.checkNotNull(email);
```

Comment:
Same as above

> These check should only be done when using the AuthenticationMethod Global-Key, because the email is not needed when using the Bearer Token (could be set to null, although an empty string is working fine)